blablas/ep4/index.md

```diff
@@ -1,16 +1,57 @@
 ---
 date: 2018-12-26
-time: 20h
+time: 20h14
 duration: "1:24:32"
 title: "MSS with  Mohamed Zakariae El Khdime!"
-tags: []
+tags: ["tech", "mss", "cybersecurity", "entrepreneurship"]
 isNext: false
 video: "2301890400043312"
 published: true
 audio: https://anchor.fm/s/88e3048/podcast/play/8871827/https%3A%2F%2Fd3ctxlq1ktw2nl.cloudfront.net%2Fproduction%2F2019-11-8%2F37063179-48000-2-83a2df7aac338.m4a
 ---
 
 In this episode of GeeksBlabla, we discuss with Mohamed Zakariae El Khdime about his succuss story.
-[//]: # "Check this github issue on How to add Episode Notes https://github.com/DevC-Casa/geeksblabla.com/issues/23 "
 
-...
+## Guests
+
+[Mohamed Zakariae El Khdime](https://www.facebook.com/infom2z)
+
+## Notes
+
+00:01 - Who is Mohamed Zakariae El Khdime?
+
+00:03 - What is your academic background?
+
+00:06 - did you study on any public high school?
+
+00:20 - How did you change your profile from developement to security?
+
+00:22 - What is cybersecurity?
+
+00:22 - What is cybersecurity?
+
+00:25 - Why do hackers always wear hoodies? Behind the stereotype?
+
+00:26 - What is your first project in entrepreneurship?
+
+00:31 - How to start a career in cybersecurity?
+
+00:33 - How you start your startup on Dubai?
+
+00:46 - Advice for someone how will start a startup?
+
+00:46 - A person who wants to start on entrepreneurship it is required to have knowledge and Capital?
+
+00:51 - How did you manage to get the funds?
+
+00:59 - What are your future vision for your startup and what is your advice for people on will start on the cybersecurity?
```

Comment:
What are your future plans, and what kind of advice could you give to someone who wants to get into cybersecurity?